src/tests/ftest/daos_test/daos_core_test-nvme_recovery.yaml

```diff
@@ -5,6 +5,9 @@ hosts:
     - server-A
     - server-B
 timeout: 600
+#Remove this once DAOS-5134 is resolved
```

Comment:
I think we should keep the reformat in-between test cases. Maybe we have 1 or 2 cases that test DAOS-5134 (like set a device as faulty, bring it back (replace with itself) and continue to use it), but still reformat between test cases I would think (to avoid other potential errors). Thoughts? @NiuYawei

Reply:
Once DAOS-5134 fixed, I think it's not necessary to do re-format between tests anymore. The only potential issue I can see is that pool destroy could leak blob if SSD is marked as FAULTY.